### DISCUSSION TITLE
Add (heading) field to HasMany panel when using ->asPanel()

### DISCUSSION THREAD

**Discussion by sanderbaas** [laravel/nova-issues#5590] [Q&A] [answered]
A HasMany field can be set to show as a panel with `->asPanel()`. Does this mean I can also assign another field to show in that same panel? For instance a heading field with explanation. How should I do that?

I already tried setting the panel with the meta array.

```
HasMany::make(__('Templates'), 'remoteEvents', RemoteEvent::class)
                ->asPanel(),

Heading::make('<p>' .
    __('Explainer help text for templates...') .
    '</p>')
    ->withMeta([
        'panel' => __('Templates')
    ])
    ->asHtml()
    ->exceptOnForms()
    ->hideFromIndex(),
```

This doesn't work as you can see below.

![afbeelding](https://github.com/laravel/nova-issues/assets/4152895/f01c818d-d9ea-4d44-b601-aad03d03aa9a)

**Comment by crynobone** [comment, marked as answer]
`asPanel()` is an internal method and not documented to be used by developers.

**Comment by sanderbaas** [comment]
@crynobone thanks for your quick response.
